code/modules/client/preference_setup/general/01_basic.dm

```diff
@@ -142,3 +142,30 @@
 			return TOPIC_REFRESH
 
 	return ..()
+
+/datum/category_item/player_setup_item/general/basic/proc/getMinAge(var/age_min)
```

Comment:
Couldn't you have coded this as a species variable instead? And then simply write the getters as:
```
/datum/category_item/player_setup_item/general/basic/proc/getMinAge()
    var/datum/species/mob_species = all_species[pref.species]
    return mob_species.age_min
```

10x prefereable to a massive switch statement.